mm_image_processing/server.py

```diff
@@ -9,16 +9,17 @@
 import json
 import asyncio
 import websockets
-import util.image_processor as image_processor
 
 from threading import Thread, Lock
-from util.s3_img_getter import getImg
 from flask_sock import Sock
 from flask import Flask, request, make_response
 from flask_cors import CORS
-from util.image_processor import processImgFromLocal
-from util.s3_img_getter import getImg, deleteImg
 
+# Ultility functions from the util directory
+from mm_image_processing.util import image_processor
+from mm_image_processing.util.s3_img_getter import getImg
+from mm_image_processing.util.image_processor import processImgFromLocal
+from mm_image_processing.util.s3_img_getter import getImg, deleteImg
```

Comment:
Note that PEP8 recommends the following:

```
import asyncio
import json
import logging
import pickle  # only needed for testing without classification service
import random
import string
from threading import Lock, Thread

import requests
import websockets
from flask import Flask, make_response, request
from flask_cors import CORS
from flask_sock import Sock

# Ultility functions from the util directory
from mm_image_processing.util import image_processor
from mm_image_processing.util.image_processor import processImgFromLocal
from mm_image_processing.util.s3_img_getter import deleteImg, getImg
```

Also, the following are not used:
- `random`
- `string`
- `threading.Lock`
- `threading.Thread`
- `requests`
- `flask.make_response`
- `flask.request`
- `mm_image_processing.util.s3_img_getter.deleteImg`

Reply:
updated to PEP8 standards